utils/scripts/ocm/ocm.py

```diff
@@ -279,13 +279,13 @@ def update_isv_cluster_info(self):
         config_file = self.configrepo_dir + "test-variables.yaml"
         with open(config_file, "r") as fh:
             data = yaml.safe_load(fh)
-        data["MONGODB"] = {}
-        data["MONGODB"]["ORG_ID"] = self.mongo_org_id
-        data["MONGODB"]["PUB_KEY"] = self.mongo_pub_key
-        data["MONGODB"]["PRI_KEY"] = self.mongo_pri_key
-        data["CRUNCHYDB"]["PUB_KEY"] = self.crunchy_pub_key
-        data["CRUNCHYDB"]["PRI_KEY"] = self.crunchy_pri_key
-        data["COCKROACHDB"]["API_KEY"] = self.crdb_api_key
+        data["MONGO"] = {}
+        data["MONGO"]["ORG_ID"] = self.mongo_org_id
+        data["MONGO"]["PUB_KEY"] = self.mongo_pub_key
+        data["MONGO"]["PRI_KEY"] = self.mongo_pri_key
+        data["CRUNCHY"]["PUB_KEY"] = self.crunchy_pub_key
+        data["CRUNCHY"]["PRI_KEY"] = self.crunchy_pri_key
+        data["COCKROACH"]["API_KEY"] = self.crdb_api_key
         with open(config_file, "w") as yaml_file:
             yaml_file.write(yaml.dump(data, default_flow_style=False, sort_keys=False))
```

Comment:
Instead of opening the file again in write mode, will it be more efficient to open it in Read+ mode at line 280?

Reply:
I have given a try with "r+" on line 280, while dumping the values on 290, the values are appending at the end of the line, instead of overriding.